installers/ansible/roles/pgo-operator/files/pgo-configs/cluster-deployment.json

```diff
@@ -340,6 +344,58 @@
                                 }
                             ]
                         }
+                    },
+                    {
+                        "name": "podinfo",
+                        "downwardAPI": {
+                            "defaultMode": 420,
```

Comment:
Do we need to write to this at all? I think we'd be fine with `0400`, which in decimal is `288`

Reply:
Nevermind, I rescind my comment.